force-app/main/default/lwc/auroraProjectSiteButton/createAuroraProjectButton.html

```diff
@@ -0,0 +1,3 @@
+<template>
+    <lightning-button variant="brand" label="New Project" title="New Project" onclick={createNewAuroraProject} class="slds-m-left_x-small"></lightning-button>
+</template>
```

Comment:
Each folder `auroraSiteDesignButton` needs a component underneath it - you can't just reuse the existing `siteAuroraSystemDesign` component and attach this template to it - you need your own javascript file for this button.

I think they even require the components to have the same name as the folder, so this file needs to be renamed to `auroraProjectSiteButton.html`